### PR TITLE
Bug 1565778 Added caveats about egress policy and networkpolicy plugin

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1565778

Will verify in the BZ.

### CODE REVIEW

**Comment by bfallonf** [comment]
BZ reporter's account has shut down, but I'd say the information needed is there.

@openshift/team-documentation PTAL

File: admin_guide/managing_networking.adoc [hunk 1]
```diff
@@ -288,17 +288,20 @@ Or,
 - A pod cannot reach specified internal subnets/hosts that it should have no
 reason to contact.
 
-You can configure projects to have different egress policies. For example,
-allowing `<project A>` access to a specified IP range, but denying the same
-access to `<project B>`. Or restrict application developers from updating from
-(Python) pip mirrors, and forcing updates to only come from desired sources.
+Egress policies can be set at the pod selector-level and project-level. For
```
Comment (kalexand-rh) [inline admin_guide/managing_networking.adoc]: s/Egress policies can be set/You can set egress policies

File: admin_guide/managing_networking.adoc [hunk 2]
```diff
@@ -288,17 +288,20 @@ Or,
...
+example, allowing `<project A>` access to a specified IP range, but denying the
```
Comment (kalexand-rh) [inline admin_guide/managing_networking.adoc]: s/allowing/you can allow
no comma
s/denying/deny

File: admin_guide/managing_networking.adoc [hunk 3]
```diff
@@ -288,17 +288,20 @@ Or,
...
+same access to `<project B>`. Or restrict application developers from updating
```
Comment (kalexand-rh) [inline admin_guide/managing_networking.adoc]: s/Or/You can also
no comma
s/forcing/force
s/desired/approved (or specified, or some other adjective. Both the CCS glossary and IBM Style say "no" to desire.)

File: admin_guide/managing_networking.adoc [hunk 4]
```diff
@@ -288,17 +288,20 @@ Or,
...
+sources.
 
 [CAUTION]
 ====
 You must have the
 xref:../install_config/configuring_sdn.adoc#migrating-between-sdn-plugins[*ovs-multitenant*
 or *ovs-networkpolicy* plug-in enabled] in order to limit pod access via egress
 policy.
+
+When using the *ovs-networkpolicy* plug-in, egress policy is only compatible with one policy per project, and will not work with projects that share a network, such as global projects.
```
Comment (kalexand-rh) [inline admin_guide/managing_networking.adoc]: s/using/you use
move only before "one policy"
no comma
s/will not/does not

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of suggestions to make it more active.

**Comment by bfallonf** [comment]
Thanks, merging.

Also, the Travis CI build, but it's not appearing for some reason. Here it is:

https://travis-ci.org/openshift/openshift-docs/builds/398049516

**Pull request merged**